packages/react-dom/src/client/ToStringValue.js

```diff
@@ -35,3 +37,42 @@ export function getToStringValue(value: mixed): ToStringValue {
       return '';
   }
 }
+
+/**
+ * Returns true only if Trusted Types are available in global object and the value is a trusted type.
+ */
+let isTrustedTypesValue: (value: any) => boolean;
+// $FlowExpectedError - TrustedTypes are defined only in some browsers or with polyfill
+if (typeof trustedTypes !== 'undefined') {
```

Comment:
Lets add a check for `enableTrustedTypesIntegration` here as well, since we don't need this function if its disabled.

Going a bit further, it'd be ideal if `toStringOrTrustedType` would just export a function that only does string coercion when `enableTrustedTypesIntegration` is set to `false`, so we don't include unnecessary code in bundles built without the flag enabled.

Reply:
But this function is only used in `toStringOrTrustedType`. It shouldn't be included at all, because it will be never used and is also not exported.

Reply:
Someone can make a mistake later and bring it in. We tend to do feature flags at both call site and in the function.

The goal isn’t to ensure it’s “not included” but more to ensure that if someone calls it without a flag, it still has a reasonable behavior. Which might be throwing, returning some placeholder value, or whatever makes sense in a particular scenario.

Reply:
For example without the flag I’d just make it return false